src/site/filters/liquid.js

```diff
@@ -140,12 +140,20 @@ module.exports = function registerFilters() {
 
   liquid.filters.featureFieldRegionalHealthService = entity => {
     if (
+      entity &&
       enabledFeatureFlags[featureFlags.FEATURE_FIELD_REGIONAL_HEALTH_SERVICE]
     ) {
       return entity.fieldRegionalHealthService
         ? entity.fieldRegionalHealthService.entity
         : null;
     }
-    return entity.fieldClinicalHealthServices[0].entity;
+    return entity && entity.fieldClinicalHealthServices
+      ? entity.fieldClinicalHealthServices[0].entity
+      : null;
   };
+
+  // used to get a base url path of a health care region from entityUrl.path
+  liquid.filters.regionBasePath = path => path.split('/')[1];
+
+  liquid.filters.isContactPage = path => path.includes('contact');
```

Comment:
You might want to do `includes('/contact')`, if possible. You could pick up other pages that include the word contact, later down the road, anything you can do to make it more specific would be useful.

Reply:
that's a good idea. i'll refactor that if this happens